google/cloud/grpc_utils/completion_queue_test.cc

```diff
@@ -15,7 +15,9 @@
 #include "google/cloud/grpc_utils/completion_queue.h"
 #include "google/cloud/future.h"
 #include "google/cloud/testing_util/assert_ok.h"
-#include <gtest/gtest.h>
```

Comment:
Do not we still want to include `<gtest/gtest.h>`?

Reply:
Elsewhere we just include `<gmock/gmock.h>` (and we do here now). I do not see the point of keeping it.